app/views/layouts/ucsdlib.html.erb

```diff
@@ -5,8 +5,8 @@
 		<%= render :partial => 'layouts/twitter_card' %>
 		<link href="<%= opensearch_catalog_path(:format => 'xml', :only_url => false) %>" title="<%= application_name%>" type="application/opensearchdescription+xml" rel="search"/>
 
-        <script src="https://library.ucsd.edu/jwplayer/jwplayer.js"></script>
-        <script src="https://library.ucsd.edu/jwplayer/key.js"></script>
+        <script src="https://content.jwplatform.com/libraries/uS59kPRL.js"></script>
+        <script src="https://libraries.ucsd.edu/jwplayer/key.js"></script>

```

Comment:
Why did this change from `libraries` to `library`?

Reply:
@mcritchlow - This flip because of the testing of the self hosted jwplayer 8 key.  The old one still works.  I'll put back the library.